tasks/ThrusterTask.cpp

```diff
@@ -0,0 +1,99 @@
+/* Generated from orogen/lib/orogen/templates/tasks/Task.cpp */
+
+#include "ThrusterTask.hpp"
+
+#include <gazebo/msgs/msgs.hh>
+
+using namespace gazebo;
+using namespace rock_gazebo;
+
+ThrusterTask::ThrusterTask(std::string const& name)
+    : ThrusterTaskBase(name)
+{
+}
+
+ThrusterTask::ThrusterTask(std::string const& name, RTT::ExecutionEngine* engine)
+    : ThrusterTaskBase(name, engine)
+{
+}
+
+ThrusterTask::~ThrusterTask()
+{
+}
+
+bool ThrusterTask::configureHook()
+{
+    if (! ThrusterTaskBase::configureHook())
+        return false;
+
+    // Set gazebo topic to advertise
+    node = transport::NodePtr( new transport::Node() );
```

Comment:
The **Node::Fini()** method unadvertise **and** remove the node using a TopicManager. 
In this case should I call **Fini()** in cleanupHook right ?
If I cannot unadvertise the topic in stopHook I think I should not advertise it in startHook.